components/PledgeSignup.client.vue

```diff
@@ -165,12 +148,13 @@ const {
   isAgreeTerms,
   isAgreeMarketing,
   warningsMap,
+  bank,
```

Comment:
since we're anyways using this hook, we can also get the bank from there instead of initializing it separately